.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
- [ ] Kindly make sure your node modules files are being ignored by git currently I can see the node modules folder in your project.